lib/matplotlib/tests/test_axes.py

```diff
@@ -6322,3 +6323,17 @@ def test_autoscale_tiny_sticky():
     ax.bar(0, 1e-9)
     fig.canvas.draw()
     assert ax.get_ylim() == (0, 1.05e-9)
+
+
+@pytest.mark.parametrize('size', [size for size in mfont_manager.font_scalings
+                                  if size is not None])
+@pytest.mark.style('default')
+def test_relative_ticklabel_sizes(size):
+    matplotlib.rcParams['xtick.labelsize'] = size
+    matplotlib.rcParams['ytick.labelsize'] = size
```

Comment:
We have a slight preference for using `mpl.rcParams`.
```suggestion
    mpl.rcParams['xtick.labelsize'] = size
    mpl.rcParams['ytick.labelsize'] = size
```